lib/Zonemaster/Backend/TestAgent.pm

```diff
@@ -58,7 +58,7 @@ sub run {
     }
     $domain = $self->to_idn( $domain );
 
-    if (defined $params->{ipv4} || defined $params->{ipv4}) {
```

Comment:
Could you update this file to be consistent about spaces and tabs in the indentation?